test/analyzers/lint_analyzer/rules/rules_list/avoid_returning_widgets/examples/example.dart

```diff
@@ -42,6 +42,20 @@ Widget _getWidget() => Container();
 
 String _getString() => '';
 
+// LINT
```

Comment:
I added two `// LINT` comments (as I see above in this file) but I don't understand the purpose of this, can you please explain?... Maybe, we should add some information about creating / modifying rule tests in contribution guide?

Reply:
From my point of view the purpose of a `// LINT` comment is to allow you to understand which lines will trigger the rule without seeing the test code or rule implementation. If it's not helping, then we really need to rethink this approach.

Reply:
I thought there is some automatic mechanism for checking these comments... Now I got it: these comments have only informational purpose. I think it still good to have them but I see a couple drawbacks:
- there is no any automation to control if these comments synchronized with actual rule implementation (and new contributors can just easily forget to add such comments)
- these comments are rule configuration specific so generally they apply only to rule with default configuration

Anyway, at the moment I also do not see how to improve this, so we might rethink this in future.